src/components/RegistryInfoContent/RegistryInfoContent.test.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render, cleanup, fireEvent } from '@testing-library/react';
 
-import RegistryInfoContent from './RegistryInfoContent';
+import { RegistryInfoContent } from './RegistryInfoContent';
 
 describe('<RegistryInfoContent /> component', () => {
-  test('should render the component in default state with npm tab', () => {
-    const wrapper = mount(<RegistryInfoContent registryUrl="https://registry.verdaccio.org" scope="@" />);
-    expect(wrapper.html()).toMatchSnapshot();
+  afterEach(() => {
+    cleanup();
   });
 
-  test('should render the component in default state with pnpm tab', () => {
-    const wrapper = mount(<RegistryInfoContent registryUrl="https://registry.verdaccio.org" scope="@" />);
-    wrapper.setState({ tabPosition: 1 });
-    expect(wrapper.html()).toMatchSnapshot();
+  test('should load the component with no data', () => {
+    const { getByTestId } = render(<RegistryInfoContent registryUrl={''} scope={''} />);
+    const unorderedListOfTodos = getByTestId('tabs-el');
+    expect(unorderedListOfTodos.children.length).toBe(1);
   });
 
-  test('should render the component in default state with yarn tab', () => {
-    const wrapper = mount(<RegistryInfoContent registryUrl="https://registry.verdaccio.org" scope="@" />);
-    wrapper.setState({ tabPosition: 2 });
-    expect(wrapper.html()).toMatchSnapshot();
+  test('should load the appropiate tab content when the tab is clicked', () => {
+    const { container, getByTestId } = render(<RegistryInfoContent registryUrl={'http://localhost:4872'} scope={''} />);
+    expect(container.textContent).not.toContain('pnpm adduser --registry http://localhost:4872');
```

Comment:
if you use an object as I mentioned above you could reuse the registryURL here :-)

Reply:
done